IntegrationTests/tests_02_syscall_wrappers/test_02_blacklisted_errnos.sh

```diff
@@ -70,12 +70,8 @@ for mode in debug release; do
         else
             exit_code=$?
             # expecting illegal instruction as it should fail with a blacklisted errno
-            if [[ "$mode" == "release" ]]; then
-                assert_equal 42 $exit_code
-            else
-                assert_equal $(( 128 + 4 )) $exit_code  # 4 == SIGILL
-                grep -q blacklisted\ errno "$temp_file"
-            fi
+            assert_equal $(( 128 + 4 )) $exit_code  # 4 == SIGILL
+            grep -q blacklisted\ errno "$temp_file"
```

Comment:
Ah, sorry, the `grep` doesn’t work correctly in release mode. Can we replace the `grep` line with:

```bash
if [[ "$mode" == "debug" ]]; then
    grep -q blacklisted\ errno "$temp_file"
fi
```

Reply:
All good 😄 
I have wrapped the grep in an if clause per your request and pushed again.